tests/integration/session_management/test_reservation.py

```diff
@@ -158,12 +158,7 @@ def test___sessions_reserved___get_connections_by_instrument_type___connections_
         ]
 
 
-# AB#2546523: Fully support querying connections for shared pins
-#
-# When you reserve sessions for all sites, the session management service
-# currently only returns shared pins for the 1st site with which they are
-# associated.
-def test___sessions_reserved_with_shared_pins_all_sites___get_connections___connections_returned_for_first_matching_site(
+def test___sessions_reserved_with_shared_pins_all_sites___get_connections___returns_connections_for_all_sites(
```

Comment:
The recent changes in the Pin map framework and follow-up updates in the session management and pin map service have enabled the reserve API to return _all_ the pins connected on an instrument's channel.